search/retrieval-augmented-generation/examples/question_answering.ipynb

```diff
@@ -47,37 +48,39 @@
         "\n",
         "Users may wish to:\n",
+        "\n",
         "1. Get an answer to a question from document(s)\n",
         "2. Get an answer to a question from document(s) along with the citation/sources\n",
         "3. Get an answer to a question from document(s) and follow it up with additional questions like in a conversation.\n",
-        "4. Users might want to change the default prompt of langchain while using the results of Enterprise Search\n",
+        "4. Users might want to change the default prompt of langchain while using the results of Vertex AI Search\n",
         "\n",
-        "In each case we will use an example of the langchain [retriever](https://python.langchain.com/docs/modules/data_connection/retrievers.html) interface to fetch contextual documents & segments from a Google Enterprise Search engine, and the Vertex AI API to call Google's PaLM2 model for summarization & answering. \n",
+        "In each case we will use an example of the langchain [retriever](https://python.langchain.com/docs/modules/data_connection/retrievers.html) interface to fetch contextual documents & segments from a Vertex AI Search engine, and the Vertex AI API to call Google's PaLM2 model for summarization & answering.\n",
         "\n",
         "Please note that the prompts and examples demonstrated here will need to be customised in order to work well with your own data.\n",
         "\n",
         "---\n",
         "\n",
         "In this notebook the following examples will be elaborated:\n",
         "\n",
-        "- ✅ Example of using `EnterpriseSearchRetriever` with langchain `RetrievalQA`\n",
-        "- ✅ Example of using `EnterpriseSearchRetriever` with langchain `RetrievalQAWithSourcesChain`\n",
-        "- ✅ Example of using `EnterpriseSearchRetriever` with langchain `ConversationalRetrievalChain`\n",
-        "- ✅ Examples of changing the default prompt with Enterprise Search and `RetrievalQAWithSourcesChain`\n",
+        "- ✅ Example of using `GoogleVertexAISearchRetriever` with langchain `RetrievalQA`\n",
+        "- ✅ Example of using `GoogleVertexAISearchRetriever` with langchain `RetrievalQAWithSourcesChain`\n",
+        "- ✅ Example of using `GoogleVertexAISearchRetriever` with langchain `ConversationalRetrievalChain`\n",
+        "- ✅ Examples of changing the default prompt with Vertex AI Search and `RetrievalQAWithSourcesChain`\n",
         "\n",
         "---\n",
         "\n",
         "**References:**\n",
         "\n",
         "- [LangChain Conceptual Documentation](https://docs.langchain.com/docs/)\n",
-        "- [LangChain Python Documentation](https://python.langchain.com/en/latest/)\n"
+        "- [LangChain Python Documentation](https://python.langchain.com/en/latest/)\n",
+        "- [LangChain Vertex AI Search Retriever Documentation](https://python.langchain.com/docs/integrations/retrievers/google_cloud_vertex_ai_search)\n"
```

Comment:
Broken link.

Reply:
This link won't work until https://github.com/langchain-ai/langchain/pull/10513 goes through

Reply:
Ah got it!